src/App/App.js

```diff
@@ -1,28 +1,73 @@
 import React, { Component } from 'react';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
 import MovieGrid from '../MovieGrid/MovieGrid.js';
 import MovieView from '../MovieView/MovieView.js';
-import ApiCalls from '../ApiCalls.js'
+import ApiCalls from '../ApiCalls.js';
+import Login from '../Login/Login.js';
 import './App.css';
 
 
-
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      user: {id: 1, name: "Olivia", email: "alan@turing.io"}
+      user: {},
+      isOnHomePage: true
+      //we will want these to be empty strings eventually...
+    }
+  }
+
+  updateCurrentUser = (userInfo) => {
+    console.log("App pre state", this.state)
+    this.setState(userInfo)
```

Comment:
Don't forget that setState expects to be given an object that specifies which part of state is being updated. By passing it the userInfo object, you may lose other parts of state you don't intend to overwrite, such as isOnHomepage